src/components/KonnectorTile.jsx

```diff
@@ -18,6 +19,17 @@ const KonnectorTile = ({
     ? konnector.categories.map(c => t(`category.${c}`))
     : []
   const subtitle = categories.join(', ')
+  let konnectorTileFooter = (
```

Comment:
Sorry but why not using a Component ? Something like:
```jsx
const KonnectorTileFooter = ({accountCount, hasUserError, subtitle, t}) => (
  { accountCount
    ? <div>
        {!!subtitle && <p className="item-subtitle">{subtitle}</p>}
        {hasUserError ? svgIcon('warning') : svgIcon('check')}
      </div>
    : <span className="item-subtitle-no-account">{t('connector.noAccount')}</span>
  }
)
```

I think it clarifies the `render()` method and separate concernes more. What do you think ?

Reply:
You're right, thanks 👍